_docs/tasks/security/authn-policy.md

```diff
@@ -88,6 +86,31 @@ NAME          AGE
 enable-mtls   1m
 ```
 
+Add this destination rule to config client side to use mTLS:
```

Comment:
(Please ignore if this has been covered somewhere else)

Do we need to explain the intention and scope of DR and Authn beforehand from high level?

Reply:
It was briefly mentioned in the authn concept. We may want to elaborate more still, but I'd like to wait until every thing stabilize.